bin/update-estimates.sh

```diff
@@ -26,5 +16,9 @@ for target in ${targets[@]}; do
   Rscript  $target
 done
 
+# Run all stable countries
+printf "Run for all regional locations"
+Rscript R/run-region-updates.R
```

Comment:
As noted in the call my concern here is non-graceful failures in a region having knock-on impacts to other estimates. This can probably just be avoided using good defensive programming.

Reply:
I discussed this with @JAllen42 - we'll get some more defensiveness in the run-region-updates.R script